content/sdk/10.js/00.ethers/20.v6/00.getting-started.md

```diff
@@ -1,69 +1,134 @@
+
+```javascript
+async function main() {
 ```
 
-Get the network (helper function [toJSON](/sdk/js/ethers/v6/providers#tojson)):
+The main function is defined as an asynchronous function to handle asynchronous operations.
+
+### Setting up Provider
 
-```ts
-console.log(`Network: ${toJSON(await provider.getNetwork())}`);
+```javascript
+  const provider = Provider.getDefaultProvider(types.Network.Sepolia);
 ```
 
-Get the latest block number:
+- **Provider.getDefaultProvider**: Initializes a provider to interact with the zkSync Sepolia network.
+
+### Fetching Network and Block Number
 
-```ts
-console.log(`Block number: ${await provider.getBlockNumber()}`);
+```javascript
+  const network = await provider.getNetwork();
+  const blockNumber = await provider.getBlockNumber();
+  console.log(`Network: ${JSON.stringify(network)}`);
+  console.log(`Block number: ${blockNumber}`);
 ```
 
-Get the block by hash (helper function [toJSON](/sdk/js/ethers/v6/providers#tojson)):
+- **provider.getNetwork()**: Fetches details about the current network.
+- **provider.getBlockNumber()**: Fetches the latest block number on the network.
+- Logs the network details and block number to the console.
 
-```ts
-console.log(`Block: ${toJSON(await provider.getBlock("b472c070c9e121ba42702f6c322b7b266e287a4d8b5fa426ed265b105430c397", true))}`);
+### Handling Transactions
+
+```javascript
+  try {
+    const transactionHash = "0xaa065e5a57e1f8470a6f258e2b6eee87c547eab066b8620ce7f3fd51405665e1";
 ```
 
-Get the transaction by hash (helper function [toJSON](/sdk/js/ethers/v6/providers#tojson)):
+- Sets a constant `transactionHash` representing a specific transaction.
+
+### Fetching Transaction Details
+
+```javascript
+    try {
+      const result = await provider.getTransactionDetails(transactionHash);
+      if (result) {
+        console.log(`Transaction Details from zkSync: ${JSON.stringify(result)}`);
+        
+        console.log(`ethCommitTxHash: ${result.ethCommitTxHash}`);
+        console.log(`ethExecuteTxHash: ${result.ethExecuteTxHash}`);
+        console.log(`ethProveTxHash: ${result.ethProveTxHash}`);
+      } else {
+        console.log(`Transaction with hash ${transactionHash} not found in zkSync.`);
+      }
+    } catch (error) {
+      console.error(`Error fetching transaction details from zkSync: ${error}`);
+    }
+```
 
-```ts
-console.log(`Block: ${toJSON(await provider.getTransaction("0x9af27afed9a4dd018c0625ea1368afb8ba08e4cfb69b3e76dfb8521c8a87ecfc"))}`);
+- **provider.getTransactionDetails(transactionHash)**: Fetches details of the specified transaction.
+- If transaction details are found, logs the details and specific Ethereum proof hashes (ethCommitTxHash,
+ethExecuteTxHash, ethProveTxHash).
+- If no details are found, logs that the transaction was not found.
+- Catches and logs any errors encountered during fetching transaction details.
+
+### Error Handling
+
+```javascript
+  } catch (error) {
+    console.error(`Error: ${error}`);
+  }
```

Comment:
This is another example of code that is too broken up. A catch can't stand on it's own, it needs to be related to a try but I don't know where this goes in relation to the code mentioned earlier.